index.js

```diff
@@ -333,6 +334,7 @@ function session(options) {
             defer(next, err);
           }
 
+          debug('session saved');
```

Comment:
this should probably be placed in conjunction with the saving log, as the session can be saved from many different places

Reply:
done, moved to `save` method.